Client/Frontend/Notifications/NotificationRootViewController.swift

```diff
@@ -88,8 +97,15 @@ extension NotificationRootViewController {
 
 // MARK: - Notification API
 extension NotificationRootViewController {
-    func showStatusNotification() {
+    func showStatusNotification(duration duration: NotificationDuration?) {
```

Comment:
In my case, when we are displaying the result of the sync for 4 seconds, I'm not sure this will work. Would I have to dismiss the `syncing...` message, causing it to animate away, and then animate it back in with the new message for 4 seconds?